crates/rust/src/lib.rs

```diff
@@ -1248,6 +1248,15 @@ fn wasm_type(ty: WasmType) -> &'static str {
         WasmType::I64 => "i64",
         WasmType::F32 => "f32",
         WasmType::F64 => "f64",
+        WasmType::Pointer => "*mut ::core::ffi::c_void",
```

Comment:
could this use `*mut u8` to be a bit easier to type and enable `.add` with a byte size directly?